setup.py

```diff
@@ -22,6 +22,6 @@
       author_email='benjamin.rueth@tum.de',
       license='LGPL-3.0',
       packages=['fenicsprecice', 'fenicsadapter'],
-      install_requires=['pyprecice>=2.0.0', 'fenics', 'scipy', 'numpy>=1.13.3'],
+      install_requires=['pyprecice>=2.0.0', 'fenics', 'scipy', 'numpy>=1.13.3', 'mpi4py'],
```

Comment:
See above. `from fenics import MPI` might be sufficient.

Reply:
We need `mpi4py` for the non-blocking communication commands, hence keeping this requirement.